examples/http-certification/custom-assets/README.md

```diff
@@ -180,11 +161,11 @@ fn create_asset_response(
     ];
     headers.extend(additional_headers);
 
-    HttpAssetResponse {
-        status_code: 200,
-        headers,
-        body: Cow::Borrowed(body),
-    }
+    HttpResponse::builder()
+        .with_status_code(200)
+        .with_headers(headers)
+        .with_body(body)
+        .build()
```

Comment:
there is one more remaining mention of `HttpAssetResponse` in line 174 below.

Reply:
Ah nicely spotted, thank you 🙏